client/css/style.css

```diff
@@ -2211,6 +2219,18 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	position: relative;
 }
 
+@media screen and (orientation: landscape) {
+	/* Pad the form with safe area insets. */
+	#form {
+		padding-bottom: env(safe-area-inset-bottom)
+	}
+
+	/* When the form becomes focused, remove the safe area insets padding. A bit of a hack, but seems to work. */
+	#form:focus-within {
```

Comment:
why exactly is this needed now? Your first version didn't have it?

Reply:
Without this, the keyboard keeps the safe area padding when it's focused, which looks weird and reduces the limited amount of vertical space. 

Here's what it looks like with this CSS (notice the extra padding in the input box):

<img width="422" alt="Padded Form" src="https://github.com/user-attachments/assets/cda5317b-c4f7-47d8-b8a3-9efb6b0b98b1">

And here it is with the CSS (notice the padding is not there, and there's more room for the channel content):

<img width="422" alt="Unpadded Form" src="https://github.com/user-attachments/assets/e1c388c1-4657-4d5f-bd11-b307c8b304be">

I'm not sure why it wasn't present in my earlier version. I tried a bunch of different ways to address this but this was the best one I landed on.

Reply:
@brunnre8 if this is a blocker (or we just want to kick this particular decision down the road) I could make one of the following changes:

1. I can remove the `#form:focus-within` hack, and we can just let the padding exist as shown in the screenshot above.
2. I can descope this pull request to support the safe area insets _only_ for portrait orientation, and leave landscape orientation as is (it can always be tackled later since portrait mode is the bigger issue, I think).

Let me know if you have a preference.